docs/set_up_transfer_and_export_device.rst

```diff
@@ -0,0 +1,230 @@
+
+|screenshot of the menu to create a new partition in the Disk Utility
+application|
+
+Give the partition on your *Transfer Device* a descriptive name
+like “Transfer Device” and select the options as in the following screenshot:
+
+|screenshot of passphrase selection prompt in the Disk Utility
+application|
+
+As noted earlier, we recommend storing this passphrase in the password manager
+for every user who will copy files using the *Transfer Device*, not in KeePassX.
+Because users will have to type in this passphrase every time they mount the
+*Transfer Device* in Tails, we recommend using a `diceware <https://en.wikipedia.org/wiki/Diceware>`__
+passphrase.
+
+After typing in the passphrase, click **Format** to continue. The Disks utility
+will ask you if you are sure: click **Format** to continue. After a few seconds,
+your new *Transfer Device* should be ready for use. If you haven't already, make
+sure to label it.
+
+.. |Disk Utility icon| image:: images/icons/disk-utility.png
+.. |screenshot of the Applications menu in Tails, highlighting Disk Utility| image:: images/screenshots/applications_accessories_disk-utility.png
+.. |screenshot of Disk Utility application| image:: images/screenshots/disk-utility.png
+.. |screenshot of the menu to create a new partition in the Disk Utility application| image:: images/screenshots/create-partition.png
+.. |screenshot of passphrase selection prompt in the Disk Utility application| image:: images/screenshots/create-passphrase.png
+
+.. [#] Tails screenshots were taken on Tails 3.0.1. Please make an issue on
+       GitHub if you are using the most recent version of Tails and the
+       interface is different from what you see here.
+
+Create a USB *Export Device*
+----------------------------
+We recommend using a fully encrypted USB drive for copying files off the
+*Secure Viewing Station*. This is even more important than for the
+*Transfer Device*, as the risk of accidentally leaving decrypted files on the
+*Export Device* is significant.
+
+Because the *Export Device* will need to be mounted on both Tails and the
+journalist's everyday workstation, you will need to use an encryption scheme
+that works on both operating systems.
+
+This is the case for many hardware-encrypted USB drives. When considering
+a hardware solution, we recommend selecting a vendor that has fully opened
+the source code and specifications of their devices and encouraged third party
+audits.
+
+VeraCrypt is a good alternative to hardware-based encryption. It is actively
+maintained cross-platform software that has been independently audited and is
+free to use.
+
+VeraCrypt-encrypted media can be opened in the Tails operating system and on
+common Linux distributions without installing additional software. To open
+VeraCrypt media on Windows or Mac workstations, or to create VeraCrypt drives,
+you need to install the VeraCrypt software. The `guide by Freedom of the Press
```

Comment:
It might be worth being explicit here and saying that you need to create the Veracrypt volumes off Tails. It's possible that someone would read this as saying you should install additional software on Tails to create Veracrypt volumes. We don't want to encourage this IMO. Users shouldn't be customizing their Tails distros without good reason.

Reply:
Added a note to the effect that it should not be installed on the SecureDrop drives in 60cc802531e97019bdc76c1723a0dde6eff66494. (IMO another Tails drive may be a fine choice.)